CONTRIBUTING.md

```diff
@@ -27,62 +26,61 @@ Been here before? Already know what you're looking for in this guide? Jump to th
-  - This is our preferred method of open communication and discussion!
-    We'd love for you to swing by to say hello.
-- Join the discussion in our [issues](https://github.com/alan-turing-institute/the-turing-way/issues) and [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls).
+- GitHub [issues](https://github.com/alan-turing-institute/the-turing-way/issues) and [pull requests](https://github.com/alan-turing-institute/the-turing-way/pulls)
+  - Join a discussion, collaborate one an ongoing task and exchange your thoughts with others.
   - Can't find your idea being discussed anywhere?
-    Open a new issue!
-    (See our [Where to start: issues](#where-to-start-issues) section below.)
-- Subscribe to our [mailing list](https://tinyletter.com/TuringWay) with which we send monthly project updates.
-- Follow [@turingway](https://twitter.com/turingway) on Twitter.
-- You can contact the Project Lead of _The Turing Way_ project - Kirstie Whitaker - by email at [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
-- You can contact the Community Manager of _The Turing Way_ project - Malvika Sharan - by email at [masharan@turing.ac.uk](mailto:masharan@turing.ac.uk).
-- You can also contact other members of the Turing Way through their preferred ways of communication [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/ways_of_working.md).
+    [Open a new issue](https://github.com/alan-turing-institute/the-turing-way/issues/new/choose)! (See our [Where to start: issues](#where-to-start-issues) section below.)
+- [Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way)
+  - This will be a good place to start if you are new to the project, or have any quick question.
+  - We'd love for you to swing by to say hello.
+- [Slack Channel](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw)
+  - For structured discussion and sustained engagement with the community members. 
+  - We will also provide notification on upcoming events and share useful resources on Slack.
+- [@turingway](https://twitter.com/turingway) on Twitter
+    - Follow us for regular updates
+- [Our mailing list](https://tinyletter.com/TuringWay) 
+    - Receive monthly project updates in a newsletter
+- Contact the Project Lead of _The Turing Way_ project - Kirstie Whitaker - by email at [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
+- Contact the Community Manager of _The Turing Way_ project - Malvika Sharan - by email at [masharan@turing.ac.uk](mailto:masharan@turing.ac.uk).
 
 ## Contributing through GitHub
 
-[Git][git] is a really useful tool for version control. [GitHub][github] sits on top of Git and supports collaborative and distributed working.
+[Git][git] is a really useful tool for version control. 
+[GitHub][github] sits on top of Git and supports collaborative and distributed working.
```

Comment:
What are `git` and `GitHub` linking to exactly? They don't quite look like Markdown heading anchors?

Reply:
Found the links at the bottom of the page!